variables.tf

```diff
@@ -5,9 +5,15 @@ variable "domain" {
 }
 
 variable "sftp_users" {
-  type        = any
-  default     = {}
-  description = "List of SFTP usernames and public keys. The keys `user_name`, `public_key` are required. The keys `s3_bucket_name` are optional."
+  type = list(
+      object({
+        user_name     = string
```

Comment:
Formatting issue here. This will cause tests to fail. Please be sure to run `terraform fmt` locally and then commit + push the result.